<launch>
    <!-- Pass parameters to the node -->

    <param name="resolution" value="0.1"/>
    <param name="world_sizeX" value="20"/>
    <param name="world_sizeY" value="20"/>
    <param name="world_sizeZ" value="4"/>

    <!-- Launch the Python script as a node -->
    <node name="bim2ros" pkg="bim2ros" type="threads.py" output="screen"/>
</launch>